### PR TITLE
Add badge for packagephobia install size

### PR DESCRIPTION
**Pull request by styfle**
Fixes [expressjs/discussions#66](https://github.com/expressjs/discussions/issues/66)

### CODE REVIEW

File: templates/badges.jade [hunk 1]
```diff
@@ -27,6 +27,10 @@ mixin badge-coverage(repo)
   a(href='https://coveralls.io/r/' + repo)
     +badge('/coveralls/c/github/' + repo + '/master')
 
+mixin badge-packagephobia(npm)
+  a(href='https://https://packagephobia.now.sh/result?p=' + npm)
```
Comment (LinusU) [inline templates/badges.jade]: Double `https://` ☺️

**Review by LinusU — APPROVED** [review]
Nice 🙌

File: templates/badges.jade (hunk 1 above)
Comment (styfle) [reply templates/badges.jade]: @LinusU Nice catch! Copy/paste fail 😄
I fixed it!

**Comment by dougwilson** [comment]
Sorry I didn't catch this before the comments. Please PR the badgeboard itself  instead of our one instance https://github.com/repo-utils/badgeboard

**Comment by styfle** [comment]
@dougwilson Done! https://github.com/repo-utils/badgeboard/pull/34

**Comment by dougwilson** [comment]
Thanks @styfle . Merged it and updated the copy of the expressjs badgeboard.

**Pull request closed without merging**